WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -135,6 +137,20 @@ private extension PointOfSaleDashboardViewModel {
                 }
             }
             .assign(to: &$isExitPOSDisabled)
+
+        totalsViewModel.$paymentState
+            .map { paymentState in
+                switch paymentState {
+                case .processingPayment,
+                        .cardPaymentSuccessful:
+                    return true
+                case .idle,
+                        .preparingReader,
+                        .acceptingCard:
+                    return false
+                }
+            }
+            .assign(to: &$isTotalsViewFullScreen)
```

Comment:
Observation:

There are errors that appear on full screen and there are errors that appear in half screen depending on the state when they appear (processing payment vs preparing reader).

Given that, we should either show errors in a consistent view-form or be mindful of that and make sure that our state/error views work both in full screen and half screen

![image](https://github.com/woocommerce/woocommerce-ios/assets/4062343/80a576d4-ff1a-42a8-92e1-2d24c45d0d0e)

Reply:
Tnx! Better error handling will be done in next PRs!